### PR TITLE
RT-2599 Fix the authorization error issue in Passenger

### PR DESCRIPTION
**Pull request by slashd0t**
changes:
 - change the run time behavior from async to synchronous.

### CODE REVIEW

**Comment by jaygel179** [comment]
can you provide details on what are the code changes all about?

`RT-2599 Fix the authorization error issue in Passenger` doesn't explain what you've done.

File: lib/stackify/utils/methods.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ def self.rounded_current_time
   end
 
   def self.is_mode_on? mode
-    Stackify.configuration.mode == Stackify::MODES[:both] || Stackify.configuration.mode == mode
+    Stackify.configuration.mode = mode || Stackify::MODES[:both]
```
Comment (jaygel179) [inline lib/stackify/utils/methods.rb]: what does `::` do?

is it accessing static attribute from `Stackify` class and with `Hash` value?
sorry for the question. no idea how ruby still works. still reading and learning it :)

Comment (slashd0t) [reply lib/stackify/utils/methods.rb]: `::`  this is a namespace operator the same symbol being use in PHP. Yes, you can access the Stackify class with the MODES property.

**Comment by slashd0t** [comment]
> can you provide details on what are the code changes all about?
> 
> `RT-2599 Fix the authorization error issue in Passenger` doesn't explain what you've done.
> 
> maybe add some PR description like:
> 
> ```
> changes:
> - remove async for some reason
> - ...
> ```
> 
> or commit related changes and explain what are the changes on your commit message

Updated the description with the commit message.

**Pull request closed without merging**